include/TPP/Transforms/Utils/VNNIUtils.h

```diff
@@ -35,16 +36,25 @@ enum class VnniOperandRank {
   BRGEMM_OUTS = 3
 };
 
-// Return the VNNI blocking factor: 2 for BF16 and 4 for BF8.
-std::optional<int64_t> getVnniBlockingFactor(Type type);
+// Return the VNNI blocking factor.
+// Optionally, operation can be provided to give access to DLTI.
```

Comment:
@rolfmorel is this the expected API usage for DLTI?

Reply:
Yes, makes sense to me. These kinds of wrappers give you a sensible default if the DLTI query they perform fails. These wrappers should take an `Operation*` to pass to `dlti::query` - if null, the query will fail/crash, so the wrapper should fallback on its default in this case as well.